indra/llappearanceutility/llbakingtexture.cpp

```diff
@@ -32,7 +32,7 @@
 #include "llgltexture.h"
 #include "llimage.h"
 
-static const bool USE_MIP_MAPS = TRUE;
+static const bool USE_MIP_MAPS = true;
```

Comment:
Also a candidate for constexpr

Reply:
I had actually forgotten to git add some files ...
Thanks for pointing that out, those booleans in a fat suite are also replaced now